onnx/onnx_cpp2py_export/defs.pyi

```diff
@@ -192,3 +192,5 @@ def get_schema(
 def get_schema(op_type: str, domain: str = "") -> OpSchema: ...
 def get_all_schemas() -> Sequence[OpSchema]: ...
 def get_all_schemas_with_history() -> Sequence[OpSchema]: ...
+def register_schema(schema: OpSchema) -> None: ...
+def deregister_schema(op_name: str, version: int, domain: str = "") -> None: ...
```

Comment:
Should we mark domain as required? It doesn't seem like a normal use case for users to deregister ops from the ONNX domain?

Reply:
Yes, I agree. It is necessary for users to explicitly assign the domain, similar to the schema's domain must be initialized explicitly during registration.